### PR TITLE
Fix Modal CSS for images and table in markdown (#739)

### PR DESCRIPTION
**Pull request by TG1999**
* Fix Modal CSS

* Address review comment

* Address review comments

---------

Date: 9th April 2024

Developer Name: Tushar Goel

---

## Issue Ticket Number

https://github.com/Real-Dev-Squad/website-status/issues/1162
## Description
Fix Modal CSS for images and table in markdown

### Documentation Updated?

- [ ] Yes
- [x] No

### Under Feature Flag

- [ ] Yes
- [x] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

Screenshot from 2024-04-09 09-22-10

</details>

## Test Coverage
<details>
<summary>Screenshot 1</summary>

</details>

### CODE REVIEW

File: task-requests/details/style.css [hunk 1]
```diff
@@ -512,6 +515,18 @@ table tr td {
   white-space: normal;
 }
 
+.requestor_description_details img {
+  width: 100%;
+}
+
+.requestor_description_details table {
+  display: block;
+  width: 100%;
```
Comment (palakgupta2712) [inline task-requests/details/style.css:524]: Do we need 100%, isn't this going to be overridden by the next width styling mentioned?

File: task-requests/details/style.css [hunk 2]
```diff
@@ -512,6 +515,18 @@ table tr td {
...
+  width: max-content;
```
Comment (palakgupta2712) [inline task-requests/details/style.css:525]: Do we need 100%, isn't this going to be overridden by the next width styling mentioned?

**Review by palakgupta2712 — CHANGES_REQUESTED** [review]

**Comment by palakgupta2712** [comment]
@TG1999 can you please add the screenshot for mobile view?

File: task-requests/details/style.css (hunk 2 above)
Comment (TG1999) [reply task-requests/details/style.css:525]: Good catch!

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**